README.md

```diff
@@ -170,7 +170,8 @@ the service worker lifecycle event you can listen for to trigger this message.
 
 For those who would prefer not to use `sw-precache` as part of a `gulp` or
 `Grunt` build, there's a [command-line interface](cli.js) which supports the
-[options listed](#options-parameter) in the API, provided via flags.
+[options listed](#options-parameter) in the API, provided via flags or an
+external JavaScript configuration file.
 
```

Comment:
Configuration via `module.exports` LGTM. I'm glad we maintain a note about the --config JSON configuration option (I personally use this in a few projects).